2022-02-24 22:36:04,737 INFO    MainThread:27492 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 22:36:04,737 INFO    MainThread:27492 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 22:36:04,737 INFO    MainThread:27492 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_223604-7l6vsrwq/logs/debug.log
2022-02-24 22:36:04,738 INFO    MainThread:27492 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_223604-7l6vsrwq/logs/debug-internal.log
2022-02-24 22:36:04,738 INFO    MainThread:27492 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc9319ce790>
2022-02-24 22:36:04,739 INFO    MainThread:27492 [wandb_init.py:init():404] calling init triggers
2022-02-24 22:36:04,739 INFO    MainThread:27492 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 22:36:04,739 INFO    MainThread:27492 [wandb_init.py:init():460] starting backend
2022-02-24 22:36:04,739 INFO    MainThread:27492 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 22:36:04,813 INFO    MainThread:27492 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 22:36:04,879 INFO    MainThread:27492 [backend.py:ensure_launched():222] started backend process with pid: 27543
2022-02-24 22:36:04,881 INFO    MainThread:27492 [wandb_init.py:init():469] backend started and connected
2022-02-24 22:36:04,922 INFO    MainThread:27492 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 22:36:04,923 INFO    MainThread:27492 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 22:36:04,923 INFO    MainThread:27492 [wandb_init.py:init():533] updated telemetry
2022-02-24 22:36:05,024 INFO    MainThread:27492 [wandb_init.py:init():563] communicating current version
2022-02-24 22:36:07,229 INFO    MainThread:27492 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 22:36:07,232 INFO    MainThread:27492 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 22:36:07,605 INFO    MainThread:27492 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 22:36:12,616 INFO    MainThread:27492 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 22:36:12,617 INFO    MainThread:27492 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 22:36:12,618 INFO    MainThread:27492 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 22:36:12,619 INFO    MainThread:27492 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 22:36:12,619 INFO    MainThread:27492 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 22:36:19,900 INFO    MainThread:27492 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-24 22:36:19,901 INFO    MainThread:27492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 22:36:56,837 INFO    MainThread:27492 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 22:36:56,839 INFO    MainThread:27492 [wandb_run.py:_restore():1752] restore
2022-02-24 22:36:57,183 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:36:57,388 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:36:57,701 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-24 22:36:57,806 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-24 22:36:57,911 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25247
}

2022-02-24 22:36:58,014 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-24 22:36:58,119 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-24 22:36:58,222 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-24 22:36:58,326 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-24 22:36:58,429 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

2022-02-24 22:36:58,950 INFO    MainThread:27492 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25247
  total_bytes: 25247
}

